generation/generate_gapic_bom.sh

```diff
@@ -17,6 +17,11 @@ for bom_directory in $(find . -maxdepth 3 -name 'google-*-bom' | sort --dictiona
   artifactId_line=$(grep --max-count=1 'artifactId' "${pom_file}")
   version_line=$(grep --max-count=1 'x-version-update' "${pom_file}")
 
+  if [[ "$groupId_line" == *"com.google.maps"* ]]; then
+    # The gapic bom includes cloud libraries
+    continue
+  fi
```

Comment:
@meltsufin This excludes the newly added library from the gapic bom (and thus from libraries-bom).